example-wasm/src/AuthenticatedApp.tsx

```diff
@@ -0,0 +1,33 @@
+import { Navigate, Outlet } from 'react-router-dom';
+
+import { Task } from './models/Task';
+import { PageLayout } from './components/PageLayout';
+
+const { RealmProvider, UserProvider, useApp } = await import('@realm/react');
+
+export function AuthenticatedApp() {
+  const atlasApp = useApp();
+  if (!atlasApp.currentUser) {
+    return <Navigate to='/' />
+  }
+
+  return (
+    <UserProvider>
```

Comment:
You know what, I see now that I can actually pass `<Navigate to='/' />` as the `fallback` prop. Thanks for some clarity 🙂 

But to add to why I didn't choose to pass e.g. the auth screen (`LoginPage`) as the fallback: For this I opted not to use the `fallback` in order to be able to show the handling of Realm and client-side routing since it can be common to provide separate routes for authenticated and unauthenticated parts, and be able to force navigate the user to any of these routes in various parts of an app.

Reply:
Done ✅  (Passed `Navigate` as the fallback)